#ifndef OBSTACLE_TRAVERSAL_BENCHMARK_OBSTACLE_TRAVERSAL_BENCHMARK_H
#define OBSTACLE_TRAVERSAL_BENCHMARK_OBSTACLE_TRAVERSAL_BENCHMARK_H

#include <ros/ros.h>

namespace obstacle_traversal_benchmark {

class ObstacleTraversalBenchmark {
public:
  ObstacleTraversalBenchmark(const ros::NodeHandle& nh, const ros::NodeHandle& pnh);
private:
};

}

#endif